lib/capistrano/tasks/composer.rake

```diff
@@ -11,10 +11,15 @@ namespace :composer do
   task :install_executable do
     on release_roles(fetch(:composer_roles)) do
       within shared_path do
+        composer_version = fetch(:composer_version, nil)
+        composer_version_option = composer_version ? "-- --version=#{composer_version}" : ""
         unless test "[", "-e", "composer.phar", "]"
```

Comment:
Generally, I don't like unless/elsif/else, as it is a little difficult to grok. Could you refactor this to be if/elsif?